chainerx_cc/examples/CMakeLists.txt

```diff
@@ -0,0 +1 @@
+add_subdirectory(mnist_cc)
```

Comment:
Shouldn't be this example named as `mnist` instead of `mnist_cc`, because this directory `chainerx_cc` is for C++ implementation? I think the existing Python example should be named as `mnist_py` instead.

Reply:
Fixed.